test/arrayops.jl

```diff
@@ -1180,7 +1179,7 @@ end
     X = [ i+2j for i=1:5, j=1:5 ]
     @test X[2,3] == 8
     @test X[4,5] == 14
-    @test isequal(ones(2,3) * ones(2,3)', [3. 3.; 3. 3.])
+    @test isequal(fill(3.,2,2), [3. 3.; 3. 3.])
```

Comment:
I wonder why this test exists under the heading "comprehensions" and what purpose it serves :).